metadata/src/main/java/edu/unc/lib/dl/util/DepositStatusFactory.java

```diff
@@ -246,8 +196,14 @@ public void clearActionRequest(String depositUUID) {
 	public void expireKeys(String depositUUID, int seconds) {
 		try (Jedis jedis = getJedisPool().getResource()) {
 			jedis.expire(DEPOSIT_STATUS_PREFIX + depositUUID, seconds);
```

Comment:
This is a minor new feature, but would you be willing to have it expire the key which gets set by the addManifest method?  I don't think those ever get removed at present.